modules/mbedtls/CMakeLists.txt

```diff
@@ -11,6 +11,18 @@ zephyr_interface_library_named(mbedTLS)
       message(WARNING "No entropy device on the system, using fake entropy source!")
     endif()
 
+    if(CONFIG_MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG_ALLOW_NON_CSPRNG)
+      message(WARNING "
+        Warning: non cryptographycally secure sources are enabled for psa_generate_random().
```

Comment:
I think no need for `Warning:` here because CMake will already indicate that.

Reply:
Yeah, but I saw [somewhere else](https://github.com/zephyrproject-rtos/zephyr/blob/c008cbab1a05316139de191b0553ab6ccc0073ad/subsys/random/CMakeLists.txt#L12) where the "Warning:" part was still added to the message so I thought I could do the same. However I don't have any strong opinion on this, so I can remove that ;)